modules/http/src/test/resources/http-listener-socket-config.xml

```diff
@@ -10,14 +10,19 @@
 
     <tcp:server-socket-properties name="globalTcpProperties" keepAlive="true" receiveBacklog="1024"
                                   receiveBufferSize="1024" reuseAddress="true" sendBufferSize="1024"
-                                  sendTcpNoDelay="true" serverTimeout="1000" linger="1000" />
+                                  sendTcpNoDelay="true" serverTimeout="500" linger="1000" />
```

Comment:
extract the timeout values to system properties. Make the sleeps relative to those values.